examples/hello_world/module/rust/src/lib.rs

```diff
@@ -33,11 +33,11 @@ struct Node {
 const STORAGE_NAME: &[u8] = b"HelloWorld";
 const FIELD_NAME: &[u8] = b"last-greeting";
 
-#[cfg(test)]
-fn init_logging() {
-    oak_tests::init_logging();
-}
-#[cfg(not(test))]
+// #[cfg(test)]
+// fn init_logging() {
+//     oak_tests::init_logging();
+// }
+// #[cfg(not(test))]
```

Comment:
Remove rather than comment-out for preference (and then might as well ding/inline `init_logging` altogether)

Reply:
Done